### DISCUSSION TITLE
Trackball

### DISCUSSION THREAD

**Discussion by Gao0130** [Schievel1/dactyl_manuform_r_track#8] [Q&A] [answered]
不好意思，我程序方面不行，正在考虑委托别人，编写带改建功能的固件，目前需要测试国产Arduino micro是否能使用，能不能提供一下你使用的烧录固件。不知道我表达清楚了没有，也就是后缀.hex的文件。

**Comment by Schievel1** [comment, marked as answer]
[handwired_tractyl_manuform_5x6_right_arduinomicro_default.hex.zip](https://github.com/Schievel1/dactyl_manuform_r_track/files/10203045/handwired_tractyl_manuform_5x6_right_arduinomicro_default.hex.zip)

**Reply by Gao0130** [reply]
Thank you mate
